src/pages/Run/Run.utils.ts

```diff
@@ -23,7 +23,7 @@ export function cleanParametersMap(params: DecodedValueMap<TaskSettingsQueryPara
  * @param viewType Possible viewType string from url
  */
 export function hasViewTypeParam(viewType?: string): viewType is string {
-  return !!(viewType && ['dag', 'timeline', 'task'].indexOf(viewType) > -1);
+  return !!viewType;
```

Comment:
`Boolean(viewType)` is easier for me to read

Reply:
Agreed. Also this function is not very useful anymore since we can't check allowed views here due to change that plugins can have any name.